test/integration/serviceaccount/external_jwt_signer_test.go

```diff
@@ -77,6 +111,11 @@ func TestExternalJWTSigningAndAuth(t *testing.T) {
 	})
 	defer tearDownFn()
 
+	// Verify that OIDC discovery docs are available.
+	if err := validateOIDCDiscoveryAvailable(restConfig); err != nil {
```

Comment:
just inline the checks here for simplicity, we're not reusing `validateOIDCDiscoveryAvailable` elsewhere

you can also make REST requests of arbitrary paths without needing to re-parse the config and urls:

`c.client.CoreV1().RESTClient().Get().AbsPath(...).Do(context.TODO()).StatusCode(&code)`

Reply:
Done, Thanks!